lib/tasks/update_sensor_types.rake

```diff
@@ -0,0 +1,21 @@
+namespace :sensorstory do
+  desc "Add min, max and fractionDigits to certain sensor types"
+  task :update_sensortypes => :environment do
+    SensorType.transaction do
+      SensorType.where(unit: '?...?').update_all(unit: nil)
+      SensorType.where(unit: '1-10').update_all(unit: nil)
+      SensorType.where(unit: '1-100').update_all(unit: nil)
+      SensorType.where(unit: '0-unlimited').update_all(unit: nil)
+      SensorType.where(unit: '').update_all(unit: nil)
+      SensorType.find_by!(property: 'Temperature').update_attributes!(min: -10.0, max: 45.0, fractionDigits: 1, unit: '°C')
+      SensorType.find_by!(property: 'pH Value').update_attributes!(min: -10.0, max: 45.0, fractionDigits: 1, unit: '')
+      SensorType.find_by!(property: 'pH Value').update_attributes!(min: 0.0, max: 14.0, fractionDigits: 1, unit: '')
+
+      #SensorType.find_by('Temperature-Humidity-Index').update_attributes!(min: 1, max: 100, fractionDigits: 0)
+      SensorType.find_by!(property: 'Milchmenge').update_attributes!(min: 0.0, max: 80.0, fractionDigits: 1, unit: ' Liter')
+      SensorType.find_by!(property: 'Movement').update_attributes!(min: 0, max: 100, fractionDigits: 0, unit: '%') # or should it be "Pedometer"?
```

Comment:
are `Movement' and 'Humidity' the right sensor types? See the comments for alternatives